Formula/parquet-tools.rb

```diff
@@ -0,0 +1,17 @@
+class ParquetTools < Formula
+  desc "ApacheParquet command-line tools and utilities"
+  homepage "https://parquet.apache.org/"
+  url "http://central.maven.org/maven2/org/apache/parquet/parquet-tools/1.8.1/parquet-tools-1.8.1.jar"
+  sha256 "9a22210a89cc03b0c4d07d34b150f0f8af4bc9c59e4344e96b966fac0e27c1fd"
+
+  bottle :unneeded
+
+  def install
+    libexec.install "parquet-tools-#{version}.jar"
+    bin.write_jar_script libexec/"parquet-tools-#{version}.jar", "parquet-tools"
+  end
+
+  test do
+    shell_output "parquet-tools cat -h"
```

Comment:
Also, can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!